packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/clientcreds/setup-app.md

```diff
@@ -0,0 +1,4 @@
+4. Click **Next**.
+5. Specify the app integration name, then click **Save**.
+
+From the **General** tab of your app integration, save the generated **Client ID** and **Client Secret** values to implement your authorization flow.
```

Comment:
same comment as above on this

Reply:
For the Client Credential grant type, there is only 5 steps in the App Integration wizard